### ISSUE TITLE
[REQUEST] Show several network interfaces simultaneously

### ISSUE THREAD

**Issue by scherepanov** [aristocratos/btop#789]
**Is your feature request related to a problem? Please describe.**
Have several network interfaces on servers. Want to watch as many as possible on same screen.
Would be interesting to be able to arbitrarily group several network interfaces and display total group stats

**Describe the solution you'd like**
Multiply network windows
Controls works only one one window. Interfaces selected individually in network window.
I am OK with any other GUI.

**Describe alternatives you've considered**
Run several btop in several small terminal windows tiled to show all interfaces.

**Additional context**
Servers has 4 x 25G interfaces and 4 x 1G interfaces (as well loopback)

**Comment by nejlontacsko** [comment]
Yep. That would be awesome, I also need that function. For usecases like load balancing.
And for embedded development observign CAN, USB, UART would be also useful.

**Comment by Ryushin** [comment]
I believe also having an option to show all summary of traffic from all NIC interfaces combined except for the loopback.
